### PR TITLE
Fix github and latestVersion links

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Fix github and latestVersion links.

### CODE REVIEW

**Review by lisad — APPROVED** [review]
SGTM. Not all use cases for data transfer move the data odd the source server but I think 'from' captures that anyway

File: index.html [hunk 1]
```diff
@@ -32,7 +33,7 @@
     <section class="informative">
       <h2>Introduction</h2>
       <p>
-        Data portability is the ability of a user to move their data on one
+        Data portability is the ability of a user to move their data from one
```
Comment (bumblefudge) [inline index.html:36]: i think this edit was probably intended for the other PR (#9) and begs the same question Lisa had there about "system" versus "instance of any supporting server"

Comment (bumblefudge) [inline index.html:36]: maybe this edit was intended for #8?

Comment (dmitrizagidulin) [reply index.html:36]: It was, yeah. ignore.

**Comment by dmitrizagidulin** [comment]
Minor non-normative change, approved, merging.

**Pull request merged**